coreppl/src/coreppl-to-mexpr/mcmc-lightweight/compile.mc

```diff
@@ -296,15 +296,6 @@ lang MExprPPLLightweightMCMC =
     TmLet { t with inexpr = exprCps env k t.inexpr }
   | TmLet ({ body = TmDist _ } & t) ->
     TmLet { t with inexpr = exprCps env k t.inexpr }
-  | TmLet ({ body = TmDist (d & { dist = DWiener w })} & t) ->
```

Comment:
This code should remain. It is there to handle CPS transformation of Wiener processes which needs to choose between a CPS or non-CPS version at runtime.